app/Models/Group.php

```diff
@@ -427,21 +427,8 @@ public function getMaxUpdatedAtDevicesUpdatedAtAttribute()
 
     public function getAutoApproveAttribute()
     {
-        // A group's events are auto-approved iff all the networks that the group belongs to are set to auto-approve
-        // events.
-        $autoapprove = false;
-
-        $networks = $this->networks;
-
-        if ($networks && count($networks)) {
-            $autoapprove = true;
-
-            foreach ($networks as $network) {
-                $autoapprove &= $network->auto_approve_events;
-            }
-        }
-
-        return $autoapprove;
+        // Events are auto-approved based on environment configuration
+        return env('FEATURE__AUTO_APPROVE_EVENTS', false);
```

Comment:
If `FEATURE__AUTO_APPROVE_EVENTS` is false, do we want to keep the previous behavior?